### DISCUSSION TITLE
Warning: Text content did not match. Server: "Hallo" Client: "greeting"

### DISCUSSION THREAD

**Discussion by JuliusMoehring** [sergiodxa/remix-i18next#92] [Q&A]
Hi, I am struggling to get i18next working properly, and it's probably a mistake by myself, but I can't figure it out.

I have followed the `README.md` closely, but for some reason, the client-side hydration seems not to be working properly.

The markup sent by the server is translating everything correctly, but as soon as the hydration starts, the correct translation is converted into the translation key. 

Does anybody have an idea, of what my config/setup issue is? Happy to share some code when you tell me, which files to check/code to share.

Thanks!

**Comment by JuliusMoehring** [comment]
For some reason, it worked suddenly... not sure why, but I'm ok with it 😆

**Comment by binajmen** [comment]
So, I had the exact same issue. I had to empty the cache and hard reload to fix mine. Hope this can spare 10 minutes of someone else in the future 😉

**Comment by nicolastoulemont** [comment]
Had the same issue when creating new locale json files, I suspect it has to do with the fact the locales files are located in the public folder 🤔 
Hard reloading + empty cache in the browser solved it as well.

**Comment by mdrahiem** [comment]
I think its caching issue. When I open "network" tab, check "Disable cache" and reload again, it works
